front/components/vaults/VaultResourcesList.tsx

```diff
@@ -267,22 +332,38 @@ export const VaultResourcesList = ({
             />
           </div>
         )}
-        {!isSystemVault && category === "managed" && (
+        {!isSystemVault && isManaged && (
           <EditVaultManagedDataSourcesViews
             owner={owner}
             vault={vault}
             systemVault={systemVault}
             isAdmin={isAdmin}
           />
         )}
-        {(category === "folder" || category === "website") && (
-          <EditVaultStaticDataSourcesViews
-            owner={owner}
-            plan={plan}
-            vault={vault}
-            category={category}
-            dataSources={dataSources}
-          />
+        {isStatic && (
+          <>
+            <EditVaultStaticDatasourcesViews
+              isOpen={showFolderOrWebsiteModal}
+              setOpen={setShowFolderOrWebsiteModal}
+              owner={owner}
+              vault={vault}
+              dataSources={dataSources}
+              dataSourceView={selectedDataSourceView}
+              plan={plan}
+              category={category}
+              onClose={() => {
+                setShowFolderOrWebsiteModal(false);
+                setSelectedDataSourceView(null);
```

Comment:
do we need to unset datasource view here ? this make the form disappear while closing, not very nice (which is also the advantage of having two different states for open and the dsv)

Reply:
The problem is that we need to reset the dsv when we want to create a new website. The other way would be to pass the a callback to `EditVaultStaticDatasourcesViews` to set the dsv to null when creating a new website. But in this case the form will disappear while opening I guess